HelpMyStreetFE/HelpMyStreetFE/Views/Pages/PrivacyPolicy.cshtml

```diff
@@ -78,13 +78,6 @@
                         <li>professional details (e.g. career history, professional memberships)</li>
                         <li>contact details (e.g. name, job title, contact number, email address)</li>
                     </ol>
-                    <p>For street champions, volunteers offering help and anyone registering an account with us:</p>
```

Comment:
Privacy Policy is Laura's domain, but do we just want to remove the reference to street champions rather than this whole section?